solvers/chop.py

```diff
@@ -59,8 +61,11 @@ def set_objective(self, X, y, lmbd):
 
         device = torch.device(self.device)
 
-        self.X = torch.tensor(X).to(device)
-        self.y = torch.tensor(y > 0, dtype=torch.float64).to(device)
+        scaler = StandardScaler()
+        X = scaler.fit_transform(X)
```

Comment:
this should not be done in the solver but in the dataset as it changes the problem

Reply:
Can it be an option in the dataset?

Reply:
the libsvmdata package does some preprocessing like this. I would just offer the good dataset (normalized or not).

Reply:
Right, this was the cause of the infinity values... (Since the scaler wasn't applied at eval time) Modifying.

Reply:
@GeoffNN dont change the solver but the dataset. Otherwise solvers will not be comparable

Reply:
Done; I added standardization as a parameter in Covtype, so it runs on both version of the dataset. We can also just keep the standardized version if you prefer.